projects/sandbox/datagen/datagen/scripts/timeslide_waveforms.py

```diff
@@ -47,7 +47,60 @@ def main(
     verbose: bool = False,
 ):
     """
-    Generates the waveforms for a single segment
+    Generates the waveforms for a single segment.
+
+    Args:
+        start:
+            GPS time of the beginning of the testing dataset
+        stop:
+            GPS time of the end of the testing dataset
+        shifts:
+            A list of shifts in seconds. Each value corresponds to the
+            the length of time by which an interferometer's timeseries is
+            moved during one shift. For example, if `ifos = ["H1", "L1"]`
+            and `shifts = [0, 1]`, then the Livingston timeseries will be
+            advanced by one second per shift, and Hanford won't be shifted
+        background:
+            Directory containing background data to use for PSD
+            calculation. Should have data for each interferometer in
+            the format generated by `background.py`
+        spacing:
+            The spacing between signals in seconds
+        buffer:
+            The buffer to apply to each side of the segment in seconds
+        prior:
+            A function that returns a Bilby PriorDict when called
+        minimum_frequency:
```

Comment:
Could be useful to provide a little more color on what these frequencies (minimum and reference) represent. I would say link to the relevant lal docs, but not sure they do a lot to elucidate this either

Reply:
I added some description, but tbh I'm not totally clear on everything that `reference_frequency` affects